bindings/java/src/main/native/Makefile.am

```diff
@@ -61,8 +61,8 @@ package:
 	$(MVNCMD) package -DskipTests
 
 # Maven install phase
-install-data-hook:
-	$(MVNCMD) install -DskipTests
+install-data-hook: package
+	cp $(java_build_dir)/jucx-@VERSION@.jar @libdir@
```

Comment:
best to use ```cp -fp ...``` to preserve permissions and force copy to support consequtive install and override.

Reply:
Done, thanks.